compiler/parser/RdrHsSyn.hs

```diff
@@ -2381,39 +2414,17 @@ mkUpdate = mkAppTy (mkQualType "Update")
 partiesType :: LHsType GhcPs
 partiesType = noLoc $ HsListTy noExt $ mkQualType "Party"
 
-mkAppTyArgs :: LHsType GhcPs -> [Located RdrName] -> LHsType GhcPs
```

Comment:
I forgot to delete a bunch of code when dropping support for generic templates v1.